src/sentry/integrations/slack/notifications.py

```diff
@@ -11,10 +12,37 @@
 from sentry.types.integrations import EXTERNAL_PROVIDERS, ExternalProviders
 from sentry.utils import json, metrics
 
+from .views.unlink_team import SUCCESS_UNLINKED_MESSAGE
+
 logger = logging.getLogger("sentry.notifications")
 SLACK_TIMEOUT = 5
 
 
+class SlackNotifyBasicMixin(NotifyBasicMixin):  # type: ignore
+    def send_message(self, channel_id: str, message: str) -> None:
+        client = SlackClient()
+        token = self.metadata.get("user_access_token") or self.metadata["access_token"]
+        headers = {"Authorization": f"Bearer {token}"}
+        payload = {
+            "token": token,
+            "channel": channel_id,
+            "text": message,
+        }
+        try:
+            client.post("/chat.postMessage", headers=headers, data=payload, json=True)
+        except ApiError as e:
+            message = str(e)
+            if message != "Expired url":
+                logger.error("slack.slash-notify.response-error", extra={"error": message})
+        return
+
+    def notify_remove_external_team(self, external_team: ExternalActor, team: Team) -> None:
```

Comment:
You can actually move this to `NotifyBasicMixin` and it should just work.

Reply:
done 👍